tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
+    "extends": "tsconfigs/browser-module",
     "compilerOptions": {
+        "esModuleInterop": false,
```

Comment:
If we let `tsconfigs` set this to `true`, we end up with a slightly different CommonJS output. [See this](https://stackoverflow.com/a/56348146/359072).